### ISSUE TITLE
"twitter:image" does not fall back to product gallery image

### ISSUE THREAD

**Issue by dariaknl** [Yoast/wpseo-woocommerce#587]
### How can we reproduce this behavior?

1. How can we reproduce this behavior?
2. Install and activate Yoast SEO 14.0 RC4.
3. Install and activate Woocommerce 4.0.1. 
4. Install and activate SEO Woo 13.0 RC2.
5. Create new product, add price and image in Product gallery:
6. Publish it and check` og:image` tags:

picture from Product gallery is picked up:
 

```
<meta property="og:image" content="http://local.wordpress.test/wp-content/uploads/misspiggy.jpg" />
<meta property="og:image:width" content="1280" />
<meta property="og:image:height" content="720" />
```
7. Go to SEO - Social - Facebook and disable Open Graph meta data.
8. Check product output from step 6:
`twitter:image` is not output. I would expect Product gallery image to be shown in `twitter:image.`

### Technical info
* WordPress version: 5.4
* Yoast SEO version: 14.0 RC4
* SEO Woo 13.0 RC2.

**Comment by jdevalk** [comment]
Is this actually a regression? I kind of doubt that it is...

**Comment by dariaknl** [comment]
No, it is not a regression, it is new functionality which we have implemented in Yoast SEO 14.0: the fallback for `twitter:image` when OG is disabled. That is why the issue has label "fix before release"
